File: langhuan-service/src/main/java/com/langhuan/common/TokenExpiredException.java
package com.langhuan.common;

/**
 * Token过期异常
 */
public class TokenExpiredException extends BaseException {
    
    public TokenExpiredException(String message) {
        super(ResponseCodeEnum.UNAUTHORIZED, message);
    }
    
    public TokenExpiredException() {
        super(ResponseCodeEnum.UNAUTHORIZED, "Token已过期，请重新登录");
    }
}

File: langhuan-server/src/main/kotlin/com/langhuan/common/TokenExpiredException.kt
package com.langhuan.common

/**
 * Token过期异常
 */
class TokenExpiredException : BaseException {
    
    constructor(message: String) : super(ResponseCodeEnum.UNAUTHORIZED, message)
    
    constructor() : super(ResponseCodeEnum.UNAUTHORIZED, "Token已过期，请重新登录")
}
